tpl/hunger.twig

```diff
@@ -24,5 +24,18 @@
         {% include 'hunger_menu.twig' with {'title': 'StuCafé Mechanical Engineering', 'week': stucafeMaschbauWeek} only %}
     </div>
 </div>
+<div class="row" style="margin-top:10%">
+    <div class="one-half column">
+        <h3>Pida Booth at ESO parking lot</h3>
```

Comment:
I think this should be "Pita Booth"?

Reply:
Probably. 
Unless it's the name of the booth? Can you confirm this @zepatrik?

Reply:
Nah, just rename it to pita, that's fine.